tests/admin_changelist/tests.py

```diff
@@ -979,12 +979,12 @@ def test_pagination_page_range(self):
         per_page = cl.list_per_page = 10
 
         for page_num, objects_count, expected_page_range in [
-            (0, per_page, []),
-            (0, per_page * 2, list(range(2))),
-            (5, per_page * 11, list(range(11))),
-            (5, per_page * 12, [0, 1, 2, 3, 4, 5, 6, 7, 8, '.', 10, 11]),
-            (6, per_page * 12, [0, 1, '.', 3, 4, 5, 6, 7, 8, 9, 10, 11]),
```

Comment:
`'.'` should be replaced with the constant `DOT` (or rather `ELLIPSIS` after the requested change above)